src/main/java/teammates/ui/webapi/output/CourseData.java

```diff
@@ -9,16 +9,16 @@
 
     private final String courseId;
     private final String courseName;
-    private final String creationDate;
     private final String deletionDate;
     private final String timeZone;
+    private final long creationDate;
```

Comment:
Called is as `creationTimestamp`? If you are going to change deletionDate, it should also be called as `deletionTimestamp`.